tests/hats_import/catalog/test_file_readers.py

```diff
@@ -8,14 +8,14 @@
 from hats.catalog import TableProperties
 
 from hats_import.catalog.file_readers import (
-    CsvPyarrowReader,
     CsvReader,
     FitsReader,
     IndexedCsvReader,
     IndexedParquetReader,
     ParquetReader,
     get_file_reader,
 )
+from hats_import.catalog.file_readers.csv import CsvPyarrowReader
```

Comment:
Why doesn't this one get imported at the `hats_import.catalog.file_readers` level too?

Reply:
Fixed